charts/mariadb/README.md

```diff
@@ -0,0 +1,101 @@
+# Clear Linux* OS `mariadb` Helm Charts
+
+<!-- Required -->
+## What is this image?
+
+`clearlinux/mariadb` is a Docker image with `mariadb` running on top of the
+[official clearlinux base image](https://hub.docker.com/_/clearlinux). 
+
+<!-- application introduction -->
+
+> [Mariadb](https://mariadb.com/) is a community-developed, commercially supported 
+> fork of the MySQL relational database management system (RDBMS), intended to remain 
+> free and open-source software under the GNU General Public License. Development is led 
+> by some of the original developers of MySQL, who forked it due to concerns over its 
+> acquisition by Oracle Corporation
+
+For other Clear Linux* OS
+based container images, see: https://hub.docker.com/u/clearlinux
+
+## Why use a clearlinux based image?
+
+<!-- CL introduction -->
+
+> [Clear Linux* OS](https://clearlinux.org/) is an open source, rolling release
+> Linux distribution optimized for performance and security, from the Cloud to
+> the Edge, designed for customization, and manageability.
+
+Clear Linux* OS based container images use:
+
+- Optimized libraries that are compiled with latest compiler versions and
+  flags.
+- Software packages that follow upstream source closely and update frequently.
+- An aggressive security model and best practices for CVE patching.
+- A multi-staged build approach to keep a reduced container image size.
+- The same container syntax as the official images to make getting started
+  easy. 
+
+To learn more about Clear Linux* OS, visit: https://clearlinux.org.
+
+<!-- Required -->
+
+## Prerequisites
+
+- PV provisioner support in the underlying infrastructure
+
+## Installing the Mariadb Chart
+
+Download the mariadb chart and install the chart with the release name ` my-release` 
+
+```
+$ helm install --name my-release mariadb
```

Comment:
Have you tried this on Clear Linux?
Before install, any init need to be done in advance?

Reply:
I have tried this and it ran successfully!
For the second question, after you install helm on clearlinux, what you need is just the command above and you don't need any other instructions!